packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1143,13 +1145,17 @@ abstract class TestRunner {
           ARBITRARY_SEQUENCE_NUMBER,
           SnapshotVersion.MIN,
           SnapshotVersion.MIN,
-          expected.resumeToken
+          byteStringFromString(expected.resumeToken)
         )
       );
       expect(actualTarget.query).to.deep.equal(expectedTarget.query);
       expect(actualTarget.targetId).to.equal(expectedTarget.targetId);
       expect(actualTarget.readTime).to.equal(expectedTarget.readTime);
-      expect(actualTarget.resumeToken).to.equal(expectedTarget.resumeToken);
+      // actualTarget's resumeToken is a hexadecimal string, but the serialized
+      // resumeToken will be a base64 string, so we need to convert it back.
+      expect(actualTarget.resumeToken || '').to.equal(
```

Comment:
See my comment below - can you verify that we are indeed using hexadecimal strings?

Reply:
As discussed offline, these are all base64 encoded strings. Thanks for updating the comment.